src/main/java/gui/MainFrame.java

```diff
@@ -29,11 +27,12 @@
 
 	/**
 	 * The mainframe constructor receives an excel table, generated from the given
-	 * excel file, as well as a list containing all the rules, stored in the main
+	 * excel file, as well as a list containing all the rules and quality indicators, stored in the main
 	 * controller
 	 * 
 	 * @param excelTable
 	 * @param rulesList
+	 * @param indicatorsQuality

```

Comment:
this isn't in the constructor, where did this come from?

Reply:
Isso é do código original do Diego. Como o Lino tá a apontar pra master, vamos ver as alterações do Diego e do Lino juntas. Ou O Lino aponta esse PR pra branch do Diego (não tem problema, qdo PR do Diego for pra master, é só apontar esse pra master de novo), ou temos que tentar perceber as diferenças xD

Reply:
Anyway, sim, esse param agora já não existe.